packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -413,17 +320,45 @@ class _HourControl extends StatelessWidget {
 /// A passive fragment showing a string value.
 class _StringFragment extends StatelessWidget {
   const _StringFragment({
-    @required this.fragmentContext,
-    @required this.value,
+    @required this.textStyle,
+    @required this.timeOfDayFormat,
   });
 
-  final _TimePickerFragmentContext fragmentContext;
-  final String value;
+  final TextStyle textStyle;
+  final TimeOfDayFormat timeOfDayFormat;
+
+  String _stringFragmentValue(TimeOfDayFormat timeOfDayFormat) {
+    String stringFragmentValue;
```

Comment:
nit: this is not needed, can just return the string within each case

Reply:
true, removed